react/components/HomeCarousel/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+
+import Type from '../../styles/Type';
+
+const Slide = styled.div`
+  text-align: center;
+  transition: opacity .5s ease-in-out;
+  opacity: ${props => (props.className.includes('slick-center') ? 1 : 0.3)};
+`;
+
+const Inner = styled.div`
+  padding: 3rem;
+  position: relative;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`;
+
+const Copy = styled.p`
+  font-size: ${Type.base};
+  width: 70%;
+  line-height: 1.5;
+  text-align: left;
+`;
+
+const Headline = styled.h2`
+  font-weight: normal;
+  margin-bottom: 1rem;
+`;
+
+const Image = styled.img`
+  max-height: 450px;
+  width: auto;
+  padding-bottom: 1rem;
+  user-drag: none;
+`;
```

Comment:
So wait: there's styling here *and* as CSS? Is that necessary?

Reply:
Those are styles that are included with `react-slick`. The styles here are specifically for the contents of the carousel.

Reply:
I see: OK, so I'd just get rid of the react-slick styles? Doesn't seem necessary from here?

Reply:
No they are (at least some of them are). I will extract them, format and get rid of the unnecessary ones.